src/routes/register.js

```diff
@@ -39,22 +39,22 @@ router.post('/', async (req, res) => {
 
 		const token = new Token({ _userId: user.uid, token: crypto.randomBytes(16).toString('hex') });
 		await token.save();
-		var transporter = nodemailer.createTransport({ 
-			service: '"Mailjet"', 
-			auth: { 
-				user: `${MAILER_USER}`, 
-				pass: `${MAILER_PASS}` 
-			} 
+		var transporter = nodemailer.createTransport({
+			service: '"Mailjet"',
+			auth: {
+				user: `${MAILER_USER}`,
+				pass: `${MAILER_PASS}`
+			}
 		});
-		var mailOptions = { 
-			from: `${ADMIN_EMAIL}`, 
-			to: user.email, 
-			subject: 'TechnoJam Account Verification', 
-			text: 'Team TechnoJam Welcomes You,\n\n' + `Your Verification Code: ${token.token}\n\nPlease verify your account by clicking the link and submitting the verification code: \nhttp:\/\/` + req.headers.host + '\/api\/verification\/' + token.token + '.\n\n'  + 'Thank You,\nTeam TechnoJam'
+		var mailOptions = {
+			from: `${ADMIN_EMAIL}`,
+			to: user.email,
+			subject: 'TechnoJam Account Verification',
+			text: 'Team TechnoJam Welcomes You,\n\n' + `Your Verification Code: ${token.token}\n\nPlease verify your account by clicking the link and submitting the verification code: \nhttp:\/\/` + req.headers.host + '\/api\/verification\/' + token.token + '.\n\n' + 'Thank You,\nTeam TechnoJam'
 		};
 		transporter.sendMail(mailOptions, (err) => {
 			if (err) { return res.status(500).send({ msg: err.message }); }
-			res.status(201).json({msg: 'Account Created! Please check your mail to confirm account.'});
+			res.status(201).json({ code: 'REGISTER_CREATE_DONE', msg: 'Account Created! Please check your mail to confirm account.' });
```

Comment:
Looks odd! Make this something else like `USER_CREATED` or `SIGNUP_SUCCESS` or something like that.